dpe/src/dpe_instance.rs

```diff
@@ -143,7 +143,7 @@ impl DpeInstance {
     ) -> Result<Response, DpeErrorCode> {
         let command = self.deserialize_command(cmd)?;
         let resp = match cfi_launder(command) {
-            Command::GetProfile => Ok(Response::GetProfile(
+            Command::GetProfile(_) => Ok(Response::GetProfile(
                 self.get_profile(&mut env.platform, env.state.support)?,
             )),
```

Comment:
Now this command is shaped like the others, you can let the command handle it internally.

```suggestion
            Command::GetProfile(cmd) => cmd.execute(self, env, locality),
```